.circleci/config.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       NODE_DEBUG: gh-pages
     steps:
       - *attach_workspace
-      - run: yarn lerna run build:demo --concurrency=1 # prevent out-of-memory
```

Comment:
I've removed this temporarily to compare speeds with styleguidist

Reply:
Please restore before merge :wink: